webpack.config.js

```diff
@@ -78,8 +78,8 @@ module.exports = {
         '/calendar/app',
         '/linagora.esn.resource/api'
       ],
-      //target: 'http://localhost:8080',
-      target: 'https://dev.open-paas.org',
+      target: 'http://localhost:8080',
+      // target: 'https://dev.open-paas.org',
```

Comment:
We should not change this

Reply:
This was a WIP to test on local. I reverted it